intake/catalog/local.py

```diff
@@ -91,10 +92,9 @@ def describe(self):
         desc = {
             'name': self.name,
             'description': self.description,
-            'type': self.type,
-            'default': self.default
+            'type': self.type or 'unknown',
```

Comment:
Does this allow the passing of arbitrary types?

Reply:
Yes, that's the idea - if the type is not set, there is no coersion